content/_index.md

```diff
@@ -7,4 +7,5 @@ undergraduates at the
 [UBC Department of Computer Science](https://www.cs.ubc.ca/). We provide a
 social space where everyone’s welcome (in ICICS 021), run events, and represent
 your interests to the department. For more events you may be interested in, please follow us
-on our social media outlets.
\ No newline at end of file
+on our social media outlets. <b>Join us on <a id="discord-serverhttpsdiscordggxf3wbydubf" href="https://discord.gg/xF3WbYDubF"> Discord</a>!</b>
```

Comment:
Should we linebreak here? It would add some space // distinguish this as the CTA.